tests/cases/fourslash/esModuleInteropFindAllReferences.ts

```diff
@@ -6,7 +8,7 @@
 ////}
 
 // @Filename: /b.ts
-////import * as a from "a";

```

Comment:
Changed this to be testing what #21703 fixed.

Reply:
Heh. I had this open in multiple editors with both versions in each to directly compare where their execution differed. I did not notice that the wrong version had been saved 🐈 Thanks.